src/GafferBindings/CompoundNumericPlugBinding.cpp

```diff
@@ -54,6 +56,23 @@ namespace
 {
 
 template<typename T>
+std::string maskedCompoundNumericPlugRepr( const T *plug, unsigned flagsMask )
+{
+	std::string extraArgs = "";
+
+	const IECore::GeometricData::Interpretation interpretation = plug->interpretation();
+	if( interpretation != IECore::GeometricData::None && interpretation != IECore::GeometricData::Color )
```

Comment:
My concern about this approach is that we potentially add that `Color` case back in at some point (because something else might want to use OpenImageIOAlgo and needs it) and that then changes the behaviour of our plugs. That would be a little odd, wouldn't it? Feels like blocking it explicitly as in my first suggestion would be more future-proof even if a little more ugly, but I'm happy to do whatever you consider the better solution :)